CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
     https://github.com/RevenueCat/purchases-flutter/pull/285
 - Removed all references to JCenter (More on the [JCenter Shutdown](https://blog.gradle.org/jcenter-shutdown))
     https://github.com/RevenueCat/purchases-flutter/pull/285
+- Updated JSON serialization using Freezed
```

Comment:
should this be under a new section?

Reply:
this was missing from the 3.9.0 changelog, but it was a part of that release

Reply:
ah okay, think i got confused because this PR does some of the releasing steps? you're going to do a different PR for the release though?

Reply:
lol yeah, I guess I got a bit ahead of myself and updated some files prematurely. I'm going to make a different PR for the release 👍